wire-ios-request-strategy/Sources/Payloads/Processing/PayloadProcessing+Conversation.swift

```diff
@@ -176,10 +195,8 @@ extension Payload.Conversation {
                                                         created: &created)
 
         conversation.conversationType = .group
-        conversation.remoteIdentifier = conversationID
```

Comment:
looks like this has been removed completely

Reply:
@David-Henner  true but the method `fetchOrCreate` called before does it already - https://github.com/wireapp/wire-ios-mono/blob/97afbdbdedd2193a8dc7558a3acf1fd47687cc91/wire-ios-data-model/Source/Model/Conversation/ZMConversation%2BCreation.swift#L63